### PR TITLE
Feature/email handler

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

**Comment by pypingou** [comment]
To merge after https://github.com/fedora-infra/summershum/pull/7 as this work is based on it

File: summershum/core.py [hunk 1]
```diff
@@ -3,10 +3,29 @@
...
+    email_handler = logging.handlers.SMTPHandler(
+        mailhost=config.get('summershum.mailhost', 'localhost'),
+        fromaddr=config.get(
+            'summershum.mailfrom', 'noreply.summershum@fedoraproject.org'),
+        toaddrs=config.get(
+            'summershum.mailto',
+            ['ralph@fedoraproject.org', 'pingou@fedoraproject.org']),
+        subject='[summershum exception]',
+        secure=None)
+    email_handler.setLevel(logging.DEBUG)
```
Comment (ralphbean) [inline summershum/core.py:26]: If this is hardcoded to `logging.DEBUG`, does that mean it will email us every log message?

Comment (ralphbean) [inline summershum/core.py:26]: Also, I think we can do this entirely from `/etc/fedmsg.d/logging.py` without any code changes.

Furthermore, we could deploy it across all of FI to all fedmsg code that way.  :)

Comment (pypingou) [inline summershum/core.py:26]: `logging.DEBUG` only logs debug message, `logging.INFO` would log every log message.

The idea of having it in `/etc/fedmsg.d/logging.py` sounds great! :)

Comment (ralphbean) [inline summershum/core.py:26]: Cool - then, I'm :-1: to the PR.  We can take it on in our ansible configuration.

**Pull request closed without merging**